### PR TITLE
[4.0 dev] mobile button visible on desktop

### PR DESCRIPTION
**Pull request by ravisaxena23**
Pull Request for Issue #27805

### Summary of Changes
Changes in SCSS as suggested by @infograf768 @brianteeman 
Screenshot (43)
### Testing Instructions
Check on any browser

### Expected result
mobile button should not be visible on desktop. gear button should not hide modals/alerts.

### Actual result
mobile button not visible on desktop

### CODE REVIEW

**Comment by brianteeman** [comment]
Unfortunately this breaks the toggle at the bottom of the screen.

It should look like this
image

but with this PR it looks like this
image

**Comment by richard67** [comment]
@ravisaxena23 In your description, the expected result and the actual result are the same text. This can't be. Expected result should describe what you expect and how it will be when this PR will be applied, and actual result shall descibe how it is now in J4 without your PR.

**Comment by ravisaxena23** [comment]
> @ravisaxena23 In your description, the expected result and the actual result are the same text. This can't be. Expected result should describe what you expect and how it will be when this PR will be applied, and actual result shall descibe how it is now in J4 without your PR.

I will remember it

**Comment by ravisaxena23** [comment]
> Unfortunately this breaks the toggle at the bottom of the screen.
> 
> It should look like this
> image
> 
> but with this PR it looks like this
> image

This is due to z-index I will fix it. I have a question why this blue bar not come on my laptop what I'm doing is wrong.?

File: administrator/templates/atum/scss/blocks/_sidebar.scss [hunk 1]
```diff
@@ -514,8 +514,11 @@
...
-    right: 10px;
-    z-index: $zindex-mobile-toggle;
+    z-index: $zindex-alerts;
+
+    [dir=ltr] & {
+      right: 10px;
+    }
 
     [dir=rtl] & {
       right: auto;
```
Comment (infograf768) [inline administrator/templates/atum/scss/blocks/_sidebar.scss]: `right: auto;` can be deleted

**Comment by infograf768** [comment]
The only solution I see is
1. Use again `$zindex-mobile-toggle` instead of `$zindex-alerts`
2. Modify the variable for `$zindex-alerts` as it should always be on top of everything.

we have now
`$zindex-alerts:                    1030;`

I would do
`$zindex-alerts:                    10000;`  // or more 

in `/administrator/templates/atum/scss/_variables.scss`

**Comment by infograf768** [comment]
Looks OK now. Weird as it looks like the Alert and the Gear are on the same z-index...

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on b86a8074f42a473850473d80d9dd7485f485e9b9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27818">issues.joomla.org/tracker/joomla-cms/27818</a>.</sub>

**Comment by ravisaxena23** [comment]
> Looks OK now. Weird as it looks like the Alert and the Gear are on the same z-index...

$zindex-mobile-toggle: 9999; which is present in burger button while gear button hass z index=$zindex-alerts: 1030;

**Comment by infograf768** [comment]
gear has same z-index than alert, but it works fine. I was not speaking of the burger

**Comment by infograf768** [comment]
@Quy can you test?

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b86a8074f42a473850473d80d9dd7485f485e9b9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27818">issues.joomla.org/tracker/joomla-cms/27818</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27818">issues.joomla.org/tracker/joomla-cms/27818</a>.</sub>

**Comment by infograf768** [comment]
tks

**Pull request merged**